app/src/main/java/com/jnj/vaccinetracker/reportsoverview/vaccinesoverview/screens/VaccinesOverviewFragment.kt

```diff
@@ -303,35 +326,93 @@ class VaccinesOverviewFragment : BaseFragment(),
         }
     }
 
+
+    @SuppressLint("SetTextI18n")
     private fun applyFilters(
-        vaccinesData: List<VaccineObservationDTO> = vaccinesOverviewViewModel.vaccineDTOs.value
-            ?: emptyList()
+        vaccinesData: List<VaccineObservationDTO> = vaccinesOverviewViewModel.vaccineDTOs.value ?: emptyList()
     ) {
+        val isStatic = { loc: String? -> loc.equals("static", ignoreCase = true) }
```

Comment:
in Constant.kt file you have defined constants for Static/Outreach/School. you should use these constants to keep  these strings in one place